clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -271,11 +268,13 @@
  *     consumer.assign(Arrays.asList(partition0, partition1));
  * </pre>
  *
- * The group that the consumer specifies is still used for committing offsets, but now the set of partitions will only
- * be changed if the consumer specifies new partitions, and no attempt at failure detection will be made.
+ * Once assigned, you can call {@link #poll(long) poll} in a loop, just as in the preceding examples to consume
+ * records. The group that the consumer specifies is still used for committing offsets, but now the set of partitions
+ * will only be changed if the consumer specifies new partitions in another call to {@link #assign(Collection) assign},
+ * and no attempt at failure detection will be made.
```

Comment:
NIT: Would it help to say, "no attempt is made by coordinator to detect consumer failures"?